packages/flutter_plugin_android_lifecycle/android/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     }
 
     dependencies {
-        implementation "androidx.annotation:annotation:1.1.0"
+        implementation "androidx.annotation:annotation:1.5.0"
```

Comment:
IIRC this requires all projects depending on the plugin to be using SDK 33, so could cause OOB breakage for our other plugins when it publishes. Could we do a sweep first to make sure all of our examples are using 33?

Reply:
Now included in this PR